python_package/trackcpp.i

```diff
@@ -36,6 +36,7 @@
 %include "std_string.i"
 %include "std_vector.i"
 %include "stl.i"
+%include "typemaps.i"
```

Comment:
what does this module do?

Reply:
oops... I did not noticed your msg here before. This module adds option to convert input/output function argument types.